src/effects/VST/VSTEffect.h

```diff
@@ -73,6 +76,136 @@ typedef SInt16 CFBundleRefNum;
 #endif
 #endif
 
+
+struct VSTEffectSettings
+{
+   // These are saved in the Config and checked against when loading a preset, to make sure
+   // that we are loading a Config  which is compatible.
+   //
+   int32_t mUniqueID;
+   int32_t mVersion;
+   int32_t mNumParams;
+
+   // When loading a preset, the preferred way is to use the chunk; when not present in
+   // the Config or failing to load, we fall back to loading single parameters (ID, value) pairs.
+   //
+   // It looks like a plugin might not support this (if their effFlagsProgramChunks bit is off)
+   // this is why it is made optional.
+   //
+   std::optional<wxString> mChunk;
+
+   // Fallback data used when the chunk is not available.
+   std::unordered_map<wxString, double> mParamsMap;
+};
+
+
+struct VSTEffectWrapper : public VSTEffectLink, public XMLTagHandler
+{
+   AEffect* mAEffect = nullptr;
```

Comment:
I notice there is an assignment of this member in the constructor of VSTEffect that is no longer necessary.

I suggest, you make a new preliminary commit, and rebase your branch onto that:

First rewrite the constructor of VSTEffect to eliminate as many assignments to members, as possible.  Instead, use in-class initializations where possible, otherwise member initializers in the constructor.